### PR TITLE
Scat 5133 team scale

### PR DESCRIPTION
**Pull request by dhivya-cognizant**
### Work checklist

- [ ] Unit tests added where applicable
- [ ] Route tests added for new pages
- [ ] New pages included in a11y tests
- [ ] Review and publish page updated
- [ ] UI changes look good on mobile

### Developer self-QA run statement

- [ ] I have clicked through the running application to see if all changes I made actually work.

### Does this PR introduce a breaking change?

- [ ] Yes
- [ ] No

### CODE REVIEW

**Review by csk95 — APPROVED** [review]

**Review by SahitiVemulapalli — APPROVED** [review]

File: src/main/features/requirements/controller/ca-teamScale.ts [hunk 1]
```diff
@@ -65,15 +101,19 @@ export const CA_POST_TEAM_SCALE = async (req: express.Request, res: express.Resp
   const { projectId } = req.session;
   const assessmentId = req.session.currentEvent.assessmentId;
   const dimension = req.session.dimensions;
-  const scalabledata = req.session.scaledata;
+  //const scalabledata = req.session.scaledata;
   //let data=parseInt(scalabledata);
   const scalabilityData = dimension.filter(data => data.name === 'Scalability')[0];
- if(req.body.team_option!=undefined){
+  const assessmentDetail = await GET_ASSESSMENT_DETAIL(SESSION_ID, assessmentId);
+  const Dimensionrequirements=assessmentDetail.dimensionRequirements;
```
Comment (BalwinderSinghCts) [inline src/main/features/requirements/controller/ca-teamScale.ts:108]: 1. Null check are not implemented 
2. assessmentDetail?.dimensionRequirements

File: src/main/features/requirements/controller/ca-teamScale.ts [hunk 2]
```diff
@@ -65,15 +101,19 @@ export const CA_POST_TEAM_SCALE = async (req: express.Request, res: express.Resp
...
+  const dimensionName=scalabilityData['name'];
```
Comment (BalwinderSinghCts) [inline src/main/features/requirements/controller/ca-teamScale.ts:109]: 1. Null check are not implemented

File: src/main/features/requirements/controller/ca-teamScale.ts [hunk 3]
```diff
@@ -65,15 +101,19 @@ export const CA_POST_TEAM_SCALE = async (req: express.Request, res: express.Resp
...
+  const weight=Dimensionrequirements.filter(x=>x["dimension-id"]===4)[0].weighting;
```
Comment (BalwinderSinghCts) [inline src/main/features/requirements/controller/ca-teamScale.ts:110]: 1. const weight=Dimensionrequirements?.filter(x=>x["dimension-id"]===4)?.[0]?.weighting;
2. use above line

File: src/main/features/requirements/controller/ca-teamScale.ts (hunk 1 above)
Comment (SahitiVemulapalli) [reply src/main/features/requirements/controller/ca-teamScale.ts:108]: these are implemented by dhivya in her latest commit

File: src/main/features/requirements/controller/ca-teamScale.ts (hunk 2 above)
Comment (SahitiVemulapalli) [reply src/main/features/requirements/controller/ca-teamScale.ts:109]: these are implemented by dhivya in her latest commit

File: src/main/features/requirements/controller/ca-teamScale.ts (hunk 3 above)
Comment (SahitiVemulapalli) [reply src/main/features/requirements/controller/ca-teamScale.ts:110]: these are implemented by dhivya in her latest commit

**Pull request merged**